### DISCUSSION TITLE
Guide for use

### DISCUSSION THREAD

**Discussion by coromjones97** [RandomNinjaAtk/arr-scripts#103] [Q&A]
I want to use this with Lidarr, I have it setup but there isn't anything on how to use it or how it works. If anyone can help me that would be great!

**Comment by RandomNinjaAtk** [comment]
It's a script that works in the background, if everything is setup correctly, you shouldn't have to do anything, which is the entire point.... Automate everything....

To make sure it's working, check the logs in the arr app or watch the container logging.... Each script has it's own set of logs which you can find in the ARR web interface (System -> Log Files)

**Reply by HitMannCU** [reply]
the thing is that i should be doing something wrong, because the only way i see any scripts logs is when i run them manually from within the container. If i restart the host, they don't start

**Reply by RandomNinjaAtk** [reply]
If they don't run automatically, either your conf file is specifying that.... Or you didn't follow the installation/configuration process correctly. 

Another common issue is that users don't use a proper text editor to modify the conf file, thus breaking it... 

So typically if it doesn't work it's, user-error and you have to figure out what you did to cause it. If you use the container as specified from the provider, and follow the instructions... It just works.....

An easy way to test if you broke it by modifying the config file, delete the config file and restart to see if it runs normally...
